### DISCUSSION TITLE
NERF Dust

### DISCUSSION THREAD

**Discussion by BUILDERlym** [NVlabs/instant-ngp#155] [Q&A]
I run into with those dust when training with my dataset. Below is the circumstance.
![image](https://user-images.githubusercontent.com/71302160/153011016-468591a5-e8e9-41b3-9195-6af95191a06e.png)
Its a cup, anyone can help me with it? thanks in advance.
data from a video with aabb_scale 8 and exhaustive colmap_matcher.

**Comment by pwais** [comment]
try `aabb_scale` of 1 or 2, make sure the cup is entirely within the cube, and you might have to mask out the background if you can't get any more views of the object.

**Reply by BUILDERlym** [reply]
one more question, how can I mask out the background?
Also, i shoot the video in 180 degree(in front of the cup), do I have to capture the back of the cup for 360 degree?

**Reply by pwais** [reply]
if you capture the back of the cub you'll get a much better 360.  pretend your camera is a spray paint and you are painting the object... you want to cover as many angles as possible

mask is not simple... try maybe https://github.com/facebookresearch/Detectron or predict depth and threshold on depth https://github.com/nianticlabs/monodepth2

**Reply by BUILDERlym** [reply]
thanks, I'll try

**Comment by BUILDERlym** [comment]
> try `aabb_scale` of 1 or 2, make sure the cup is entirely within the cube, and you might have to mask out the background if you can't get any more views of the object.

thanks, i will try
